otelcollector/main/main.go

```diff
@@ -0,0 +1,260 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func main() {
```

Comment:
could we clean up the logs a little bit on start up?
<img width="1440" alt="image" src="https://github.com/Azure/prometheus-collector/assets/79080068/219e1ddd-619d-469b-8840-933ba8f4d3e0">

Reply:
I see that I actually have not added debug mode to the ccp image. I have added a task to implement this across the whole thing [here](https://msazure.visualstudio.com/InfrastructureInsights/_workitems/edit/27071288). For now I have commented the regex values as its not needed and the merged file is enough for debugging.